asrtoolkit/convert_transcript.py

```diff
@@ -3,43 +3,34 @@
 Python class for converting file formats used in Automatic Speech Recognition
 """
 
-import argparse
 import logging
 import sys
 
+from fire import Fire
+
 from asrtoolkit.file_utils.script_input_validation import assign_if_valid
 
 LOGGER = logging.getLogger(__name__)
 
 
 def check_input_file_validity(input_file):
     if input_file is None:
-        LOGGER.error("Invalid input file {}".format(input_file))
+        LOGGER.error("Invalid input file %s", input_file)
         sys.exit(1)
 
 
 def convert(input_file, output_file):
+    """
+    Convert between text file formats
```

Comment:
```suggestion
    Convert between text file formats (supported formats are stm, json, srt, vtt, and html)
```

Note that fire can give the following help text
```
NAME
    convert_transcript.py - Convert between text file formats
SYNOPSIS
    convert_transcript.py INPUT_FILE OUTPUT_FILE
DESCRIPTION
    Convert between text file formats
POSITIONAL ARGUMENTS
    INPUT_FILE
    OUTPUT_FILE
NOTES
    You can also use flags syntax for POSITIONAL ARGUMENTS